x-pack/plugins/ml/public/embeddables/anomaly_swimlane/anomaly_swimlane_embeddable.tsx

```diff
@@ -101,14 +101,20 @@ export class AnomalySwimlaneEmbeddable extends Embeddable<
     super.render(node);
     this.node = node;
 
+    const I18nContext = this.services[0].i18n.Context;
+
     ReactDOM.render(
-      <ExplorerSwimlaneContainer
-        id={this.input.id}
-        embeddableInput={this.getInput$()}
-        services={this.services}
-        refresh={this.reload$.asObservable()}
-        onOutputChange={(output) => this.updateOutput(output)}
-      />,
+      <I18nContext>
+        <EmbeddableSwimLaneContainer
```

Comment:
I think we should add a few pixels padding above the swim lane. It is too close to the title and panel edit controls:

![image](https://user-images.githubusercontent.com/7405507/86238108-e1702b00-bb94-11ea-8c79-16f91a1e888f.png)

Reply:
Set padding in https://github.com/elastic/kibana/pull/70063/commits/98526c415513d76756a6be99a213ce0806ae80ac